api/tls_context.proto

```diff
@@ -41,6 +41,16 @@ message TlsCertificate {
   repeated DataSource signed_certificate_timestamp = 5;
 }
 
+message TlsSessionTicketKeys {
+  // Optional identifier for this set of keys
+  string name = 1;
```

Comment:
Should there also be a ConfigSource for kds here, similar to https://github.com/envoyproxy/data-plane-api/pull/180

Reply:
This should be `KdsSecretConfig`, but you might want to wait a day or two, before #180 is merged.

Reply:
Yeah, let's wait for #180 to get merged and then I'll fix this up.

Reply:
This is now merged, do you want to rebase/merge?